docs/troubleshooting/harvester.md

```diff
@@ -67,6 +67,121 @@ Users can generate a support bundle in the Harvester GUI with the following step
+1. Break the automatic downloading.
+
+- Click `Generate Support Bundle` from Harvester WebUI.
+
+- When the progress is about 20%, close the web page.
+
+- Check support bundle status from kubectl.
+
+```
+# kubectl get supportbundle -A -oyaml
+apiVersion: v1
+items:
+- apiVersion: harvesterhci.io/v1beta1
+  kind: SupportBundle
+  metadata:
+    creationTimestamp: "2024-02-02T11:18:09Z"
+    generation: 5
+    name: bundle-htl5f
+    namespace: harvester-system
+    resourceVersion: "1218311"
+    uid: a3776373-05fe-4584-8a9a-baac3fa91bbf
+  spec:
+    description: sp1
+    issueURL: ""
+  status:
+    conditions:
+    - lastUpdateTime: "2024-02-02T11:18:38Z"
+      status: "True"
+      type: Initialized
+    filename: supportbundle_db25ccb6-b52a-4f9d-97dd-db2df2b004d4_2024-02-02T11-18-10Z.zip
+    filesize: 8868712
+    progress: 100
+    state: ready
+```
+
+When a supportbundle object has status `progress: 100`, the support bundle file is ready to be downloaded.
+
+2. Download and retain the support bundle file.
+
+- Download from curl or wget
+
+You need to manually construct the URL with `?retain=true` param to retain the support bundle file after downloading.
+
+```
+curl -k https://192.168.122.199/v1/harvester/supportbundles/bundle-htl5f/download?retain=true -o sb2.zip
+```
+
+Check the supportbundle object, it is kept for reusing.
+
+```
+# kubectl get supportbundle -A
+NAMESPACE          NAME           ISSUE_URL   DESCRIPTION   AGE
+harvester-system   bundle-htl5f               sp1           3h43m
+```
+
+- Download from web browser
+
+Manually input the URL `https://192.168.122.199/v1/harvester/supportbundles/bundle-htl5f/download?retain=true` into the web browser.
+
+When `?retain=true` param is not added, downloading the support bundle file will lead to the deletion of related resources.
+
+3. Delete resources.
+
+Retaining the support bundle files will consume more resources. Each support bundle file is backed by a `supportbundle-manager-bundle*` pod in the `harvester-system` namespace. And the generated support-bundle zip file is kept in `/tmp` folder of the pod. A big support bundle file means the pod uses more memory resources.
+
+```
+NAME                                                    READY   STATUS    RESTARTS       AGE
+supportbundle-manager-bundle-dtl2k-69dcc69b59-w64vl     1/1     Running   0              8m18s
+```
+- Delete Manually
+
+You can delete the supportbundle object with the following command.
+
+```
+# kubectl delete supportbundle -n harvester-system bundle-htl5f
+supportbundle.harvesterhci.io "bundle-htl5f" deleted
+
+# kubectl get supportbundle -A
+No resources found
+
+```
+
+When a supportbundle object is deleted, the backed pod is deleted automatically.
+
+- Delete Automatically
+
+Harvester deletes the related resources silently based on the settings below.
+
+[support-bundle-expiration](../advanced/index#support-bundle-expiration)
+
+[support-bundle-timeout](../advanced/index#support-bundle-timeout)
```

Comment:
As I know, `support-bundle-timeout` is not related to deletion. It's the way to check support bundle manager timeout, but not actually delete it. It just stops the process.

BTW, I also wrote some [explanation here](https://github.com/harvester/harvester/issues/4922#issuecomment-1920400493).

Reply:
sure, the `expiration` is the right one.  put them two here, is to give user more information.